docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,195 @@
+
+What is the Buzzer
+====================
+Your module is capable of inducing specific vibration frequencies in order to create audible tones. These buzzer tones are used to indicate several module statuses. 
+Here, we will discuss all possible “songs” your module may play during normal operation and their meanings. We will also discuss how to use the 
+:ref:`Buzzer Control client <buzzer_control>` in order to create and play your own songs.
+
+Standard Songs
+==================
+
+Startup
+-----------
+On a successful power-up and initialization, your module will play a 5-tone startup song. After this song completes, your module is ready for use.
+
+The complete startup song is played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/startup_song.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+In the event that your module is programmed with improper firmware, its calibration data becomes corrupted, or its product key becomes corrupted your module will only 
+play a 3-tone startup song. If :ref:`reprogramming your module <updating_firmware_control_center>` does not resolve this issue, please contact us at support@vertiq.co.
+
+The failed startup song is played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/bad_startup.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Arming
+-------
+If you are using the :ref:`Advanced Arming feature <manual_advanced_arming>`, then, on arming, your module will perform its :ref:`advanced_arming_behavior` which includes the arming song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/arming_song.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+Disarming
+------------
+If you are using the :ref:`Advanced Arming feature <manual_advanced_arming>`, then, on disarming, your module will perform its :ref:`advanced_disarming_behavior`. 
+If the song playback option is configured to either Once or Continuously, you will hear the disarming song played here:
+
+.. raw:: html
+
+    <audio controls>
+        <source src="../_static/manual_images/buzzer_song_mp3s_pics/disarm.mp3" type="audio/mpeg">
+    Your browser does not support the audio element.
+    </audio>
+
+If the song playback option is set to Continuously, the disarm song will repeat until the disarm state is broken.
+
+Analog Hobby Protocol Calibration
+-------------------------------------
+While :ref:`performing analog hobby protocol calibration <hobby_calibration_tutorial>`, you will hear two separate song portions indicating the calibration's progress. 
+The first portion of the song indicates that calibration is starting. This is followed by tones played twice as fast indicating that calibration is in progress. 
```

Comment:
The first portion of the song indicates that the calibration has started and is looking for the highest possible throttle signal.  This is followed by tones played twice as fast indicating that the calibration is looking for the lowest possible throttle signal.